src/webrtc/stats/groupCallStats.ts

```diff
@@ -75,7 +75,9 @@ export class GroupCallStats {
             summary.push(c.processStats(this.groupCallId, this.userId));
         });
 
-        Promise.all(summary).then((s: Awaited<CallStatsReportSummary>[]) => this.summaryStatsReportGatherer.build(s));
+        Promise.all(summary)
+            .then((s: Awaited<CallStatsReportSummary>[]) => this.summaryStatsReportGatherer.build(s))
+            .catch((err) => alert(err));
```

Comment:
Showing an alert when the stats gatherer fails seems like it would be a confusing experience for the user. Could we just log the errors?